package.json

```diff
@@ -10,7 +10,7 @@
   "scripts": {
     "start:dev": "nodemon server/index.js",
     "start": "node server/index.js",
-    "seed": "",
+    "seed": "node server/seed.js --lines 100 --output server/listingImageData.csv",
```

Comment:
Maybe I misunderstood this, but how can you be sure that your script works correctly if you haven't written 10M records to your csv file?

Reply:
Super fair! I ended up having to refactor a couple things, so thank you for mentioning that.